projects/js-toolkit/packages/npm-bundler/src/adapt/bundler-project/index.ts

```diff
@@ -74,7 +74,7 @@ async function transformBundles(): Promise<void> {
 		const sourceFile = bundlerWebpackDir.join(fileName);
 
 		if (!fs.existsSync(sourceFile.asNative)) {
-			break;
+			continue;
```

Comment:
😱

Reply:
:shrug: We are using it in so many little builds that it didn't cause any error.